ee/dtr/index.md

```diff
@@ -6,61 +6,42 @@ keywords: registry, repository, images
 
 >{% include enterprise_label_shortform.md %}
 
-Docker Trusted Registry (DTR) is the enterprise-grade image storage solution
-from Docker. You install it behind your firewall so that you can securely store
-and manage the Docker images you use in your applications.
+Docker Trusted Registry (DTR) is Mirantis's enterprise-grade image storage solution. Installed behind the firewall, either on-premises or on a virtual private cloud, DTR provides a secure environment from which users can store and manage Docker images.
```

Comment:
I wonder if we even need to mention firewall here.  I think DTR would stand up pretty well if it wasn't behind a firewall.  I'm sure it will almost always be installed behind a firewall, but this sort of implies that it needs a firewall.  In reality, it needs to be behind load balancers not just a firewall (when operating in HA config).